Source/WebCore/rendering/RenderElement.h

```diff
@@ -300,7 +300,10 @@ class RenderElement : public RenderObject {
 
     bool layerCreationAllowedForSubtree() const;
 
-    enum StylePropagationType { PropagateToAllChildren, PropagateToBlockChildrenOnly };
+    enum class StylePropagationType {
+        PropagateToAllChildren,
+        PropagateToBlockChildrenOnly
```

Comment:
Can we get rid of `PropagateTo`? Reads a bit redundantly now that it's an enum class